src/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/StrConcat.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MartinGeorgiev\Doctrine\ORM\Query\AST\Functions;
+
+/**
+ * Implementation of PostgreSQL string concatenation (using ||).
+ *
+ * @see https://www.postgresql.org/docs/current/functions-string.html
```

Comment:
```suggestion
 * @see https://www.postgresql.org/docs/15/functions-string.html
```
Because the `current` URL evolves over time and the contents may not be available at this address after a new release of PostgreSQL goes live.